### PR TITLE
Add 24th release blog post

### PR DESCRIPTION
**Pull request by anatoliykmetyuk**

### CODE REVIEW

File: docs/blog/_posts/2020-04-28-24th-dotty-milestone-release.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+Now, the above error message also contains the following sentence:
+
+```
+   |             Maybe you are missing an else part for the conditional?
+```
+
+We hope this change will make the language more intuitive for new users.
+
+# Inline overrides
```
Comment (anatoliykmetyuk) [inline docs/blog/_posts/2020-04-28-24th-dotty-milestone-release.md:55]: @odersky @nicolasstucki could you verify if everything is alright with this piece?

File: docs/blog/_posts/2020-04-28-24th-dotty-milestone-release.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+Inline overrides are now supported. For example, consider the following code:
```
Comment (nicolasstucki) [inline docs/blog/_posts/2020-04-28-24th-dotty-milestone-release.md:56]:
```suggestion
Inline overrides and inline abstract methods are now supported. For example, consider the following code:
```

File: docs/blog/_posts/2020-04-28-24th-dotty-milestone-release.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+```scala
+abstract class A:
+  def f(x: Int) = s"Foo $x"
+
+class B extends A:
+  inline override def f(x: Int) = s"Bar $x"
+
+@main def Test =
+  val b = B()
```
Comment (nicolasstucki) [inline docs/blog/_posts/2020-04-28-24th-dotty-milestone-release.md:66]:
```suggestion
  val b: B = B()
```
Just to be explicit

File: docs/blog/_posts/2020-04-28-24th-dotty-milestone-release.md (hunk 1 above)
Comment (nicolasstucki) [reply docs/blog/_posts/2020-04-28-24th-dotty-milestone-release.md:55]: Maybe we should merge https://github.com/lampepfl/dotty/pull/8785 before the release

File: docs/blog/_posts/2020-04-28-24th-dotty-milestone-release.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+REPL now supports indented code. Consider the following snippet:
+
+```scala
+scala> if true then
+     |   print(1)
+     |   print(2)
+     |
+```
+
+Previously, the REPL would have stopped after `println(1)`. Now, it waits either for an `else` block or an extra newline to indicate the end of the expression. The above example will output `12` as expected.
```
Comment (nerush) [inline docs/blog/_posts/2020-04-28-24th-dotty-milestone-release.md:28]: `println(1)` is not used in the above snippet, it uses `print(1)`

File: docs/blog/_posts/2020-04-28-24th-dotty-milestone-release.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+Head to our [Getting Started page for new contributors](https://dotty.epfl.ch/docs/contributing/getting-started.html),
+and have a look at some of the [good first issues](https://github.com/lampepfl/dotty/issues?q=is%3Aissue+is%3Aopen+label%3Aexp%3Anovice).
+They make perfect entry points into hacking on the compiler.
+
+We are looking forward to having you join the team of contributors.
+
+## Library authors: Join our community build
+
+Dotty now has a set of widely-used community libraries that are built against every nightly Dotty
+snapshot. Currently, this includes ScalaPB, algebra, scalatest, scopt and squants.
```
Comment (milessabin) [inline docs/blog/_posts/2020-04-28-24th-dotty-milestone-release.md:101]: +shapeless

File: docs/blog/_posts/2020-04-29-24th-dotty-milestone-release.md [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+and have a look at some of the [good first issues](https://github.com/lampepfl/dotty/issues?q=is%3Aissue+is%3Aopen+label%3Aexp%3Anovice).
+They make perfect entry points into hacking on the compiler.
+
+We are looking forward to having you join the team of contributors.
+
+## Library authors: Join our community build
+
+Dotty now has a set of widely-used community libraries that are built against every nightly Dotty
+snapshot. Currently, this includes shapeless, ScalaPB, algebra, scalatest, scopt and squants.
+Join our [community build](https://github.com/lampepfl/dotty-community-build)
```
Comment (arturopala) [inline docs/blog/_posts/2020-04-29-24th-dotty-milestone-release.md:137]: This link leads to the archived repository, one has to click https://github.com/lampepfl/dotty/tree/master/community-build again.

**Pull request merged**